.gitignore

```
@@ -0,0 +1,5 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/.vs/*
```

```suggestion
.vs/
```
The * is implied for a directory and we don't need to limit to the root.